packages/templates/src/getTemplates.js

```diff
@@ -64,7 +64,11 @@ async function loadTemplate(title, imageBaseUrl) {
       const srcPath = `${imageBaseUrl}images/templates/${
         data.default.slug
       }/posters/${i + 1}`;
-      return { webp: `${srcPath}.webp`, png: `${srcPath}.png` };
+      return {
+        webp: `${srcPath}.webp`,
+        png: `${srcPath}.png`,
+        type: data.default.pages[i].pageTemplateType,
```

Comment:
🤔 Nice, is this being used anywhere yet?

Reply:
Not yet, but it will be in [9093](https://github.com/google/web-stories-wp/issues/9093) This way we can still filter the layout screenshots by type.